spec/spec_helper.rb

```diff
@@ -57,6 +57,15 @@
   config.mock_with :rspec do |mocks|
     mocks.yield_receiver_to_any_instance_implementation_blocks = true
   end
+
+  # Monkey-patch Marshal.dump to enable dumping of mocked or stubbed objects. By
```

Comment:
Should this go :point_up: with the other config.mock_with section?  I'm fine with moving the comments and adding whitespace to separate the two but I had to look at the two blocks side by side to realize they were for the same context.